### PR TITLE
Fix KeyboardAvoidView for android

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
The KeyboardAvoidView was causing issues on Android, not letting the buttons to be tapped while the keyboard was opened, by setting the KeyboardAvoidView as disabled the issue is fixed.

Also added a few missing translations in the same PR.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-12554

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Includes text changes and localization file updates

### CODE REVIEW

**Comment by hmhealey** [comment]
@enahum I thought we got rid of KeyboardAvoidingView in favour of KeyboardLayout? Regardless, KeyboardLayout doesn't really do anything on Android anyway, so disabling KeyboardAvoidingView is definitely the way to go without switching everything to KeyboardLayout

File: app/screens/login/login.js [hunk 1]
```diff
@@ -143,6 +143,14 @@ export default class Login extends PureComponent {
         Keyboard.dismiss();
         InteractionManager.runAfterInteractions(async () => {
             if (!this.props.loginId) {
+                t('login.noEmail');
```
Comment (hmhealey) [inline app/screens/login/login.js]: Can this be done in a separate PR? It's not related to the other changes

Comment (enahum) [reply app/screens/login/login.js]: yes I can move it to a separate PR

**Comment by enahum** [comment]
@hmhealey done.
